appveyor.yml

```diff
@@ -1,4 +1,7 @@
 environment:
+  # This is required for at least an AArch64 compiler in one image, and is also
+  # going to soon be required for compiling LLVM.
+  APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017 Preview
```

Comment:
Does this still need to be the `Preview` image?

Reply:
I'm not sure, but I think it's best if we change one thing at a time